src/providers/DevicesProvider/AudioInputProvider.tsx

```diff
@@ -76,7 +84,11 @@ const AudioInputProvider: React.FC = ({ children }) => {
             `Audio devices updated and "default" device is selected. Reselecting input.`
           );
           try {
-            await audioVideo?.chooseAudioInputDevice(selectedInputRef.current);
+            let currentDevice: Device | VoiceFocusTransformDevice   = 'default'
```

Comment:
Have we tested out in JS SDK meeting demo to see whether it has the same issue? I don't see any VF logic handling there so I assume so.

Reply:
Just checked in JS SDK meeting demo, we don't have this kind issue. And yes, VF does not handle this issue.

Reply:
Wait, so does JS SDK have this issue or not?

Reply:
> Wait, so does JS SDK have this issue or not?

Yep, JS SDK meeting demo does not have this issue. If the device is a default VF transform device, the `currentAudioInputDevice` in this [line](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts#L1438) won't be a string `default`, instead, it will be a VF transformed device. However, in React library, we use `meetingManager.selectedAudioInputDevice: string | null` to define the current audio input, when the current device is a default VF transformed device, it is still string `default`. I think the root issue is that, in JS SDK, we receive a real device as selected device, while in React library, we only accept `String`.